recipes/prgenv-gnu/24.7/gh200/config.yaml

```diff
@@ -1,6 +1,8 @@
 name: prgenv-gnu
 spack:
-  commit: releases/v0.22
+  # TODO: Change to releases/v0.23
+  # https://github.com/spack/spack/pull/46464
+  commit: 1fb60eaaf0c7e22a5d6cb7fdb188c3fa8fb48753
```

Comment:
To do:
```suggestion
  commit: releases/v0.23
```

Reply:
Don't we want to keep v0.22 for the 24.7 version of prgenv-gnu?

Reply:
Yeah, absolutely. I didn't mean to change 24.7, but 24.11. I've pushed the correct-er commits now and removed all 24.7 changes.